### PR TITLE
Add cpp anonymous namespace snippet

### PR DESCRIPTION
**Pull request by blukat29**
Adds 'ans' keyword for c++ anonymous namespace.

### CODE REVIEW

File: snippets/cpp.snippets [hunk 1]
```diff
@@ -138,6 +138,11 @@ snippet ns
 	namespace ${1:`vim_snippets#Filename('', 'my')`} {
 		${0}
 	} /* namespace $1 */
+# anonymous namespace
+snippet ans
+	namespace {
+		${0}
+	} /* anonymous namespace */
```
Comment (lpil) [inline snippets/cpp.snippets]: Is this commented needed?

Comment (blukat29) [reply snippets/cpp.snippets]: Which comment are you referring to? `# anonymous namespace` is not needed. But `/* anonymous namespace */` should be there at least for consistency with the snippet 'ns'.

Comment (lpil) [reply snippets/cpp.snippets]: Could we remove it from both please?

**Comment by lpil** [comment]
Thanks!

**Pull request merged**